app/controllers/follow_requests_controller.rb

```diff
@@ -1,9 +1,11 @@
 class FollowRequestsController < ApplicationController
   before_action :set_follow_request, only: %i[ show edit update destroy ]
+  before_action { authorize @follow_request || FollowRequest }
+
 
   # GET /follow_requests or /follow_requests.json
   def index
-    @follow_requests = FollowRequest.all
+    @follow_request = FollowRequest.all
```

Comment:
Careful with editing code out of context ; stay focused on authorization here.

I get that this probably did not break anything since the index route is not valid, but if you decide you want an index of the follow_requests, this will break an otherwise working feature.